src/components/Form/Form.module.scss

```diff
@@ -5,3 +5,11 @@
     margin: 0 -0.75rem;
   }
 }
+.asLink {
+  background: none !important;
+  border: none;
+  padding: 0 !important;
+}
+button.asLink {
+  font-size: 1rem;
+}
```

Comment:
I think what is being done here is now in the design system (no example yet but the class exists).

<button class="wmnds-btn wmnds-btn--link">Display like a normal link</button>